documentation/v0.2.0/example-use-cases/json_similary_search.md

```diff
@@ -0,0 +1,2587 @@
+---
```

Comment:
The format of this markdown is wrong. Seems like the online editor doesn't work properly. Please try converting your notebook by using this tutorial https://www.python-engineer.com/posts/convert-colab-markdown/